tests/h/services/search_index/service_test.py

```diff
@@ -49,20 +105,34 @@ def test_it_notifies_of_annotation_transformation(
                 AnnotationTransformEvent.return_value
             )
 
-    @pytest.mark.parametrize("target_index", (None, "another"))
     def test_it_calls_elasticsearch_as_expected(
-        self, search_index, annotation, target_index, es_client
+        self, search_index, annotation, es_client
     ):
-        search_index.add_annotation(annotation, target_index=target_index)
+        search_index.add_annotation(annotation)
 
         es_client.conn.index.assert_called_once_with(
-            index=es_client.index if target_index is None else target_index,
+            index=es_client.index,
             doc_type=es_client.mapping_type,
             body=Any(),
             id=Any(),
             refresh=False,
         )
 
+    @pytest.mark.usefixtures("with_reindex_in_progress")
+    def test_it_calls_elasticsearch_again_for_a_reindex(
+        self, search_index, annotation, es_client
+    ):
+        search_index.add_annotation(annotation)
+
+        assert es_client.conn.index.call_count == 2
+        es_client.conn.index.assert_called_with(
+            index="another_index",
+            doc_type=es_client.mapping_type,
+            body=Any(),
+            id=annotation.id,
+            refresh=False,
+        )
```

Comment:
This test is a bit wishy washy, as we don't actually specify what the body is, but... I think it's close enough